phpcs.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset>
+  <arg value="psv"/>
+  <arg name="extensions" value="php"/>
+
+  <file>src</file>
+  <file>tests</file>
+
+  <exclude-pattern>/vendor/</exclude-pattern>
+
+  <rule ref="PSR12"/>
+
+  <rule ref="PSR2.Classes.PropertyDeclaration.Underscore">
+    <exclude-pattern>/src/</exclude-pattern>
+  </rule>
+
+  <rule ref="PSR12.Properties.ConstantVisibility.NotFound">
```

Comment:
this rule is perhaps useful to enable - however not sure if it causes BC breaks

Reply:
We need to change `php` requirement in `composer.json` to `>= 7.1`

Reply:
ohh yea that requires a version bump - so maybe for later then